### DISCUSSION TITLE
Prometheus exporter for NGINX Unit metrics

### DISCUSSION THREAD

**Discussion by lcrilly** [nginx/unit#993] [Show and tell]
The NGINX Unit control API includes a `/status` endpoint for [usage statistics](https://unit.nginx.org/usagestats/). Here is a solution for exposing these metrics in Prometheus format:

https://gist.github.com/lcrilly/ad516de8378dd8ae5303a942e67d55b5

**Comment by Pavlusha311245** [comment]
In addition to the discussion, I will attach a link to this issue #837  and also make my suggestions. 
It might be worth making an endpoint/metric and also adding the ability to disable it, for example when running `unitd --prometheus` as an argument
